### ISSUE TITLE
W013 false positive

### ISSUE THREAD

**Issue by XhmikosR** [twbs/bootlint#207]
```
"bootlint: " " W013 " "Bootstrap version might be outdated. Latest version is at least 3.3.1 ; saw what appears to be usage of Bootstrap 3.3.1"
```

Bootstrap is already 3.3.1, https://github.com/mpc-hc/mpc-hc.org/commit/7cfb0c1c2c80d1efa6a909782dc77b0516f9f86a
